releases/23.10.0/O3DE 2310 Feature List

```diff
@@ -0,0 +1,58 @@
+
+# (WIP) O3DE 23.10.0 Feature List
+
+**Status:** In Progress
+
+**Open 3D Engine (O3DE)** 23.10 release levels up the authoring experience for game creators and robotics simulation developers. In addition to improvements to the core engine, robotics users in O3DE now have the ROS2 Gem, a Gem that provides integration with the [Robot Operating System (ROS2)](https://www.ros.org/), which provides drivers, state-of-the-art algorithms, and developer tools for developing robotics simulations. Game developers in O3DE now have the [Multiplayer Sample game](https://github.com/o3de/o3de-multiplayersample) available to them - a ready-to-use game that's not only fun to play, but provides developers with example implementations of common game elements to help start their own projects.
```

Comment:
nit: revise the following:
```
developer tools for developing robotics simulations.
```
to
```
developer tools for robotics simulations.
```

Reply:
What you are seeing isn't the title, but the directory for the file, which does use the 23.10.0 format in case we need to create a new directory for a point release/hotfix (ie 23.10.1).

I am only including sigs that will have content. Sig-test, sig-operations, sig-release, and a few others do not contribute code or features to a release, and are not mentioned as they will always have blank entries.